test/get.coffee

```diff
@@ -304,3 +304,28 @@ describe 'quest', ->
         quest options, (err, resp, body) ->
           assert.ifError err
           done()
+
+      it 'is thenable on success', (done) ->
```

Comment:
Thoughts on putting these inside their own describe? Like `describe "promise support" ->`? I could go either way but if it makes it more clear I think it makes sense?